### ISSUE TITLE
Doesn't mark errors in text editor for .vue files

### ISSUE THREAD

**Issue by ihodev** [jwldnr/VisualLinter#52]
### Installed product versions
- Visual Studio: 2017
- This extension: 1.1.87

### Description

Doesn't show errors in text editor for `.vue` files.

### Steps to recreate
1. Create and open `hello.vue` file
2. Paste next contents
```vue
<template>
  <div>
    Hello
  </div>
</template>

<script>
const a = 5
</script>
``` 
3. Lint

### Current behavior

![shot_190130_185140](https://user-images.githubusercontent.com/9801624/51998002-8a0ded00-24c0-11e9-9a5a-17919f5c0994.png)

### Expected behavior

![shot_190130_185140](https://user-images.githubusercontent.com/9801624/51998303-10c2ca00-24c1-11e9-890c-bb9790658bbd.png)

**Comment by jwldnr** [comment]
Thank you for the detailed report!
I’ve seen this before with vue files. Have you checked the output window for VisualLinter to see if you are getting any messages?
They can often help identify the problem.

**Comment by jwldnr** [comment]
Actually, I've pushed a new version (`1.1.88`) to the marketplace just now. Please update the extension and see if it makes any difference. 👍

**Comment by ihodev** [comment]
Just updated. Unfortunately, it's still not working. And I think there is the same output in the `ActivityLog.xml`:

```xml
  <entry>
    <record>670</record>
    <time>2019/01/31 09:32:02.397</time>
    <type>Error</type>
    <source>Editor or Editor Extension</source>
    <description>System.ArgumentException: The collections refer to different snapshots.&#x000D;&#x000A;   at Microsoft.VisualStudio.Text.NormalizedSnapshotSpanCollection.IntersectsWith(NormalizedSnapshotSpanCollection set)&#x000D;&#x000A;   at jwldnr.VisualLinter.Tagging.MessageTagger.&lt;&gt;c__DisplayClass7_0.&lt;GetTags&gt;b__0(MessageMarker marker) in C:\projects\visuallinter\src\VisualLinter\Tagging\MessageTagger.cs:line 36&#x000D;&#x000A;   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()&#x000D;&#x000A;   at Microsoft.VisualStudio.Text.Tagging.Implementation.TagAggregator`1.&lt;GetTagsForBuffer&gt;d__39.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)</description>
  </entry>
```

**Comment by jwldnr** [comment]
I am able to reproduce this, however I haven't got enough time to look at it tonight. I'll have to take a closer look this weekend.

I'll let you know how it goes.

**Comment by jwldnr** [comment]
There seem to be something wrong with the latest update I pushed, I will be reverting to the old version while I work on a fix.
Please "upgrade" your extension to version `1.1.90` which will soon be available on the marketplace.

For now, I'll close this issue but I will use it as reference to make sure future updates work as intended.
If you're still having problems after upgrading, please post here and I'll reopen the issue.

Thank you for the help!
